docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+
+A Block Storage *Volume* houses these blocks of data. Volumes can be *attached* to a cloud computing instance, which makes its data and filesystem available to the instance. If your instance is running Linux, then mounting a Volume's filesystem is just like mounting any other filesystem.
+
+Volumes are stored separately from your cloud instances, but inside the same data center, and they are attached via the data center's private networking. A Volume can be detached from a cloud instance and its data will persist, even if the cloud instance is deleted. The Volume can also be re-attached to a different instance (though only one attachment at a time is possible). Volumes can also be increased in size at any time, independent of an instance's built-in storage.
+
+### Benefits and Limitations
+
+A Block Storage Volume augments the raw storage capacity of a cloud instance, which can be useful if your storage needs are greater than your computing demands. Because a Volume is scalable, it can adapt as your data grows in size. Additionally, all data stored with Linode Block Storage is replicated three times, so your Volumes are highly available and fault tolerant.
+
+{{< note >}}
+While the health and uptime of Linode Block Storage is closely monitored by Linode Support, we still recommend [making separate backups](/docs/security/backups/backing-up-your-data/) of your Volumes.
+{{< /note >}}
+
+Because Volumes are directly connected to an instance, their I/O speeds are much faster than those of an alternative storage solution like Object Storage. As well, the nature of Block Storage allows you to read and write small parts of your data, which means that you can incrementally update your files. This is in contrast to Object Storage, which requires a full re-upload of a file to update it.
+
+Some aspects of Block Storage lead to natural limitations. In particular, a Volume needs to be attached to a cloud instance for its data to be accessible. In comparison, a file stored in Object Storage can be downloaded by any internet connected client at any time.
 
 Below you will find some of the more popular use cases for Block Storage.
 
 ## Use Cases
 
 ### Databases
 
-Databases require quick read/write operations and, with the ever increasing demands of data, the ability to scale. Block Storage Volumes are mounted directly to the filesystem, so there is a minimal delay in writing and retrieving data. Additionally, it's easy to upsize a Block Storage Volume so one can scale the Volume to match the increasing demands of a database.
+Databases require quick read/write operations, and Block Storage Volumes are mounted directly to a cloud instance's filesystem, so there is a minimal delay in writing and retrieving data. Growing businesses often have increasing data storage demands, so it's also important for a database to be scalable. A Volume that stores your database can be resized to meet your storage needs.
```

Comment:
Split this into more sentences, and changed some of the subject/verb agreement:

To me, `to match the increasing demands of a database` implies that the database has increasing data demands; to see, it's more that the business that runs the database has the data demands.